<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EMI-数据处理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/emipdf/acbel -1.jpg</normaloff>:/emipdf/acbel -1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QAction { font-size: 12px; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_input" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel { font-size: 15px; }</string>
         </property>
         <widget class="QLabel" name="label_modelName">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>120</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>机种数据路径：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_loadQTY">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>120</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>负载数量：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_tempFile">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>120</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>模板(可选)：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_name">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>80</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;填写机种数据文件夹路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="statusTip">
           <string>请填入或选择机种数据文件夹所在路径</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;填写机种数据文件夹路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="overwriteMode">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_tempFile">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自定义模板文件路径，留空则使用默认模板&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="statusTip">
           <string>请填入或选择模板路径</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自定义模板文件路径，留空则使用默认模板&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="overwriteMode">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_loadQTY">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>140</y>
            <width>421</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;程序依据此项选择对应默认模板，如果自定义模板则该项无效&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;程序依据此项选择对应默认模板，如果自定义模板则该项无效&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_pathName">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>80</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>浏 览</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_pathTemp">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>20</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>浏 览</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_button" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_deal">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>开始处理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_exit">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>退 出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionsetting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelpdoc"/>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionlog"/>
    <addaction name="actiondocx2pdf"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar_main"/>
  <action name="actionsetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionhelpdoc">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actiondocx2pdf">
   <property name="text">
    <string>docx转pdf</string>
   </property>
   <property name="toolTip">
    <string>docx转pdf</string>
   </property>
  </action>
  <action name="actionlog">
   <property name="text">
    <string>查看日志</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
